pkg/local_workflows/output_workflow.go

```diff
@@ -101,6 +102,23 @@ func outputWorkflowEntryPoint(invocation workflow.InvocationContext, input []wor
 		mimeType := input[i].GetContentType()
 
 		if strings.HasPrefix(mimeType, content_type.LOCAL_FINDING_MODEL) {
+			var local_findings local_models.LocalFinding
+			err := json.Unmarshal(input[i].GetPayload().([]byte), &local_findings)
+			if err != nil {
+				log.Warn().Err(err).Msg("Failed to unmarshal local finding")
+				continue
+			}
+			findingPresentation := presenters.LocalFindingPresentation{
```

Comment:
Suggestion: The fact that we have to add the path to an extra structure means for me, that this information is missing in the local findings model. The path is not a presenter specific concern but a test concern hence it should be in the findings model.